lisa/features/serial_console.py

```diff
@@ -135,3 +148,30 @@ def check_panic(
 
         if panics:
             raise LisaException(f"{stage} found panic in serial log: {panics}")
+
+    def check_exception(
```

Comment:
When this method is needed? Why not add patterns in kernel panic?

Reply:
Does it make the system hang? If so, it's same as kernel panic.

Reply:
No, the system doesn't hang. It will enter into 'initramfs', which is usually caused by disk corruption or the file system is not mounted properly. In the kdump issue, there is disk corruption because the disk driver is not correct, which also causes the file system check failed.

When entering into 'initramfs',  If the disk can be repaired, such as using 'fsck' to repair it, then we can input 'reboot' in the 'initramfs' model, and the system can boot up normally.

Reply:
can we rename it more accurately than "exception"? For example, rename it like "check_initramfs"? There may be better names.